Parser/readWQMdat.m

```diff
@@ -1,438 +1,440 @@
+    %                   Turbidity         ('TURB'): NTU
+    %                   Density           ('DENS'): kg/m^3
+    %                   Photosynthetically active radiation ('PAR'): umol_phtn/m^2/s
+    %
+    %                 Also contains some metadata fields. The '.dat' output
+    %                 format does not contain any calibration information, so
+    %                 only the following are present:
+    %
+    %                   instrument_make:      'WET Labs'
+    %                   instrument_model:     'WQM'
+    %                   instrument_serial_no: retrieved from input file
+    %
+    % Author:       Paul McCarthy <paul.mccarthy@csiro.au>
+    % Contributor:  Brad Morris <b.morris@unsw.edu.au>
+    % 				Guillaume Galibert <guillaume.galibert@utas.edu.au>
+    %
+    % See http://www.wetlabs.com/products/wqm/wqm.htm
+    %
+
+    %
+    % Copyright (C) 2017, Australian Ocean Data Network (AODN) and Integrated
+    % Marine Observing System (IMOS).
+    %
+    % This program is free software: you can redistribute it and/or modify
+    % it under the terms of the GNU General Public License as published by
+    % the Free Software Foundation version 3 of the License.
+    %
+    % This program is distributed in the hope that it will be useful,
+    % but WITHOUT ANY WARRANTY; without even the implied warranty of
+    % MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    % GNU General Public License for more details.
+
+    % You should have received a copy of the GNU General Public License
+    % along with this program.
+    % If not, see <https://www.gnu.org/licenses/gpl-3.0.en.html>.
+    %
+    % ensure that there are exactly two arguments
+    narginchk(2, 2);
+    if ~ischar(filename), error('filename must contain a string'); end
+
+    % Lookup arrays for supported and required fields
+    params = {};
+
+    %
+    % this table provides mappings from the WQM field name (the column header
+    % in the input file) to the IMOS compliant parameter name. It also contains
+    % comments for some parameters.
+    %
+    params{end + 1} = {'WQM', {'', ''}};
+    params{end + 1} = {'SN', {'', ''}};
+    params{end + 1} = {'WQM-SN', {'', ''}};
+    params{end + 1} = {'MMDDYY', {'', ''}};
+    params{end + 1} = {'HHMMSS', {'', ''}};
+    params{end + 1} = {'MM/DD/YY', {'', ''}};
+    params{end + 1} = {'HH:MM:SS', {'', ''}};
+    params{end + 1} = {'Cond(mmho)', {'CNDC', ''}}; % mmho <=> mS , I think /mm is assumed
+    params{end + 1} = {'Cond(S/m)', {'CNDC', ''}};
+    params{end + 1} = {'Temp(C)', {'TEMP', ''}};
+    params{end + 1} = {'Pres(dbar)', {'PRES_REL', ''}};
+    params{end + 1} = {'Sal(PSU)', {'PSAL', ''}};
+    params{end + 1} = {'DO(mg/l)', {'DOXY', ''}};
+    params{end + 1} = {'DO(mmol/m^3)', {'DOX1', ''}}; % mmol/m3 <=> umol/l
+    params{end + 1} = {'DO(ml/l)', {'DOX', ''}};
+    params{end + 1} = {'CHL(ug/l)', {'CPHL', getCPHLcomment('unknown','470nm','695nm')}}; %older files
```

Comment:
I would prefer you include all function declaration that you introduce in the same commit. This is OK if we squash but not easy to review!

Reply:
In addition this is inconsistent with other parsers where you don't use this function.

Reply:
Correct - this was not supposed to be here. 
I probably merged something when doing the rebase for the WQMs files.